doc/content/getting-started/faq/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "FAQ"
+description: ""
+weight: 5
+---
+
+Check out frequently asked question about {{% tts %}} and learn how to get the most out of your deployment.
+
+<!--more-->
+
+## Why to use LoRaWAN and {{% tts %}} for your next IoT use case?
+
+LoRaWAN is a communication protocol that has proven advantage in transmitting small sensor data packets over long distances in comparison to other wireless technologies, which makes it great for various use cases, especially industrial deployments. {{% tts %}} is a complete LoRaWAN Network Server that enables you to easily utilize LoRaWAN features to build your use case, while being fully compliant to the LoRa Alliance specifications.
```

Comment:
`a proven advantage`. Also please use the registered `®` symbol for the first instance of LoRaWAN.